### DISCUSSION TITLE
Multiline support for chat embed

### DISCUSSION THREAD

**Discussion by josh900** [FlowiseAI/Flowise#566] [Q&A] [answered]
Does anyone know if there is a way to enable multiline input in the embedded or standalone shared chatbot? Or maybe some tips on what code needs to be looked at to do this?

**Comment by HenryHengZJ** [comment, marked as answer]
the embed code is here - https://github.com/FlowiseAI/FlowiseChatEmbed
this is where you should change the [code](https://github.com/FlowiseAI/FlowiseChatEmbed/blob/main/src/components/inputs/textInput/components/ShortTextInput.tsx#L14)
